src/components/Chat/react-native-gifted-chat/src/MessageContainer.js

```diff
@@ -75,10 +75,30 @@ export default class MessageContainer extends React.Component {
     const messagesData = this.prepareMessages(nextProps.messages);
     this.setState({
       dataSource: this.state.dataSource.cloneWithRows(messagesData.blob, messagesData.keys),
-      opponentInfoInfo: nextProps.opponentInfoInfo,
+      opponentInfo: nextProps.opponentInfo,
     });
   }
 
+  getWorkInfo(user) {
```

Comment:
Is this function for getting user's experience?
Now we replaced work as experience!